src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/ConfigurationTest.php

```diff
@@ -391,6 +391,73 @@ public function testBaselineFileWriteError()
         $configuration->writeBaseline();
     }
 
+    public function testExistingIgnoreFile(): void
+    {
+        $filename = $this->createFile();
+        $ignorePatterns = [
+            '/Test message .*/',
+            '/^\d* occurrences/',
+        ];
+        file_put_contents($filename, json_encode($ignorePatterns));
+
+        $configuration = Configuration::fromUrlEncodedString('ignoreFile='.urlencode($filename));
+        $trace = debug_backtrace();
+        $this->assertTrue($configuration->isIgnoredDeprecation(new Deprecation('Test message 1', $trace, '')));
+        $this->assertTrue($configuration->isIgnoredDeprecation(new Deprecation('Test message 2', $trace, '')));
+        $this->assertFalse($configuration->isIgnoredDeprecation(new Deprecation('Test mexxage 3', $trace, '')));
+        $this->assertTrue($configuration->isIgnoredDeprecation(new Deprecation('1 occurrences', $trace, '')));
+        $this->assertTrue($configuration->isIgnoredDeprecation(new Deprecation('1200 occurrences and more', $trace, '')));
+        $this->assertFalse($configuration->isIgnoredDeprecation(new Deprecation('Many occurrences', $trace, '')));
+    }
+
+    public function testIgnoreFilePatternWarning(): void
```

Comment:
Sorry, apparently fabbot does not like `void` in test files, it's not part of the Symfony CS.

Reply:
fixed